BasicCasting/README.md

```diff
@@ -0,0 +1,142 @@
+## Integration CaaS Guide
```

Comment:
Integration Guide

This guide shows how to integrate the Google Cast SDK into your application. We provide a CaaS (Cast as a Service) object, which not only contains `setup()` and `cast()` functions, but also contains a cast event listener, allowing you to complete this feature conveniently.